crowdsec-docs/unversioned/getting_started/next_steps.md

```diff
@@ -0,0 +1,42 @@
+---
+id: next_steps
+title: Post Installation Steps
+---
+
+Now that you have installed CrowdSec, we have a post installation steps to help continue your journey.
+
+### 1. CrowdSec Console
```

Comment:
First thing could be to check the installation was successful:
* Check there are no port conflicts: 8080 for lapi and 6060 for prometheus
* Check that the bouncer and other stuff are communicating correctly with the SE
* .. any other check ?...
* Then it's a good transition to finish by testing an enroll in the console

Reply:
Console enroll is important, but is it more important than having metrics and acquisitions working ?